cmd/fs-v1-helpers.go

```diff
@@ -418,11 +409,6 @@ func fsRenameFile(ctx context.Context, sourcePath, destPath string) error {
 		return err
 	}
 
-	// Verify if source path exists.
-	if _, err := os.Stat(sourcePath); err != nil {
-		return osErrToFSFileErr(err)
-	}
-
```

Comment:
are we now ignoring the error if sourcePath is not found?

Reply:
No renameAll returns same error.